icx-cert/Cargo.toml

```
@@ -1,6 +1,6 @@
 [package]
 name = "icx-cert"
-version = "0.21.0"
+version = "0.22.0"
```

Same for all Cargo.toml, with respect to changing the version number.
```suggestion
version = "0.21.0"
```

---

Done.